### PR TITLE
[PE-4149] Add support for portal styled cards

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/PE-4149

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/542/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/544/ for more details.

File: src/chi/components/card/card.scss [hunk 1]
```diff
@@ -159,4 +159,17 @@
...
+  &.-portal {
+    background-color: set-color(grey, 20);
+    border: 0;
+    border-radius: 0;
+
+    .a-card__header {
+      align-items: center;
+      background-color: set-color(grey, 30);
+      box-shadow: none;
+      padding: 0 1rem;
```
[the remark this answers is not in the document]
Comment (mattnickles) [reply src/chi/components/card/card.scss]: @jllr In the documentation example, the Portal base `a-card__header` height should be consistent with the Portal with tabs `a-card__header` height. Applying -sm should solve that.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/552/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/553/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/564/ for more details.

**Comment by mattnickles** [comment]
@jllr [Sent a PR](https://github.com/jllr/ux-chi/pull/6) for storing card tabs outside of card header

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/570/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/572/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/577/ for more details.

**Pull request merged**